experimental/packages/opentelemetry-instrumentation-grpc/src/instrumentation.ts

```diff
@@ -89,10 +100,18 @@ import { VERSION } from './version';
 
 export class GrpcInstrumentation extends InstrumentationBase<GrpcInstrumentationConfig> {
   private _metadataCapture: metadataCaptureType;
+  private _semconvStability = SemconvStability.OLD;
 
   constructor(config: GrpcInstrumentationConfig = {}) {
     super('@opentelemetry/instrumentation-grpc', VERSION, config);
     this._metadataCapture = this._createMetadataCapture();
+
+    const entries = getStringListFromEnv('OTEL_SEMCONV_STABILITY_OPT_IN');
+    if (entries?.some(entry => entry?.toLowerCase() === 'http/dup')) {
```

Comment:
Question: Should we trim this here in case there are spaces in their value? For example, `"db,http"` works fine as expected, but `"db, http"` does not because of the added whitespace: `' http'` is not `'http'`.

Reply:
I think `getStringListFromEnv()` takes care of some of that already:

```
> process.env.FOO = 'db, http'
'db, http'
> getStringListFromEnv('FOO')
[ 'db', 'http' ]
```

I think the `?` in `entry?.toLowerCase()` is redundant too since the individual entries are guaranteed to be strings.